lunch-application/src/theme/Home.vue

```diff
@@ -32,25 +31,26 @@
           </tr>
         </thead>
         <tbody>
-          <tr v-for="row in posts">
+          <tr v-for="row in restaurantData">
```

Comment:
this table will display the exact same data as the other table. intentional, or something you're coming back to? (If it's something you're gonna circle back to, slap a TODO comment on it so it's clear it's in-progress code.)